imports/print/shared.lua

```diff
@@ -0,0 +1,44 @@
+local printLevelConvar = GetConvar('ox:printlevel', 'info')
+
+local PrintLevel = {
+    debug = {
+        level = 1,
+        prefix = '^6[DEBUG] ',
+    },
+    info = {
+        level = 2,
+        prefix = '^7[INFO] ',
+    },
+    warn = {
+        level = 3,
+        prefix = '^3[WARN] ',
+    },
+    error = {
+        level = 4,
+        prefix = '^1[ERROR] ',
+    },
+}
+
+---Prints to console conditionally based on what ox:printlevel is.
+---Any print with a level more severe will also print. If ox:printlevel is info, then warn and error prints will appear as well, but debug prints will not.
+---@param level 'debug' | 'info' | 'warn' | 'error'
+---@param message any
+function Print(level, message)
+    message = type(message) == "string" and message or json.encode(message)
```

Comment:
This should instead check for `'table'` and encode only then. Should pass options through as well (store the options as a local rather than making a new table each time).

```lua
local options = { sort_keys = true, indent = true }
```

Reply:
done.